### DISCUSSION TITLE
[Weekly Curation Report]: July 3 - 7

### DISCUSSION THREAD

**Discussion by danidi** [wikipathways/wikipathways-help#46] [curation-reports]
### Recent changes

Recent Changes: 28 pathways with recent changes. Several pathways without "under construction tag", I'm not sure if that means that they are finished: WP5360, WP5395, WP5394
Added needs work with concrete suggestions to WP5360, added add description tag to WP5367, WP5362.
Added PW ontology tags to WP5394, WP5180.
https://classic.wikipathways.org/index.php/Pathway:WP2806 had a needs work tag that many identifiers are wrong, I've added some examples that I could identify (see request from @egonw on slack). Also several DOIs don't resolve.
https://classic.wikipathways.org/index.php/Pathway:WP5389 is both approved and under construction @DeniseSl22, @egonw. 

### Under construction

Under Construction: 

Approved WP5385

@khanspers are https://classic.wikipathways.org/index.php/Pathway:WP5378 and https://classic.wikipathways.org/index.php/Pathway:WP5379 test pathways that could be deleted at some point? Or should it be kept for a tutorial?

### New pathways

New pathways: 
https://classic.wikipathways.org/index.php/Pathway:WP5394
https://classic.wikipathways.org/index.php/Pathway:WP5395
https://classic.wikipathways.org/index.php/Pathway:WP5396

### Deleted Pathways

Deleted pathways: 
https://classic.wikipathways.org/index.php/Pathway:WP5343 is tagged for deletion since 8th of May by Tooba, but without providing a reason for the deletion. I kept it for now.

### Additional notes

Created issue https://github.com/wikipathways/wikipathways-help/issues/45.

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by khanspers** [comment]
Thanks @danidi! A few comments:

- WP5395: Added the "Under construction" tag since this pathway is the topic of a discussion https://github.com/wikipathways/wikipathways-help/issues/45
- WP5394: Added "Approved" tag as it looks complete to me
- WP5378/WP5379: Deleted, just my own test pathways ;)
